[docs] Added example of integration with Graylog at log-shipper module (#11100)
deckhouse/deckhouse: modules/460-log-shipper/docs/EXAMPLES.md

    syslog.message_id: "{{ request_id }}"
```

## Graylog integration

> This example shows how to configure the integration with Graylog. Make sure that Graylog is configured to receive incoming messages over the TCP protocol on the specified port:

Author: ```suggestion
Make sure that an incoming stream is configured in Graylog to receive messages over the TCP protocol on the specified port. Example manifest for integration with Graylog:
```
